active_checks/check_sftp

```diff
@@ -21,7 +21,7 @@ OPTIONS:
   --host HOST            SFTP server address
   --user USER            Username for sftp login
   --secret SECRET        Secret/Password for sftp login
-  --private-key KEY      Private Key for sftp login
+  --private-key KEY      Path to rsa private Key file for sftp login
```

Comment:
Lets call this `--private-key-file PATH` for clarity

Reply:
makes sense to me.

I also had to adjust the whitespacing.